scripts/start.sh

```diff
@@ -54,7 +55,19 @@ echo "Creating role bindings for principals"
 docker-compose exec tools bash -c "/tmp/helper/create-role-bindings.sh"
 
 echo
-docker-compose up -d kafka-client schemaregistry replicator-for-jar-transfer connect control-center
+echo "Building custom Docker image with connect version ${CONFLUENT_DOCKER_TAG} and connector version ${CONNECTOR_VERSION}"
+if [[ "${CONNECTOR_VERSION}" =~ "SNAPSHOT" ]]; then
```

Comment:
TODO: we can later push this logic into a script which can be used at either custom-image-build-time, or even container start-time.

Currently differentiates based on matches `SNAPSHOT`, but I wonder how much work there would be to just make this process local and remote (Hub) names alike or serially.

Reply:
Can we consider this for a jira for future enhancement?